test_results_win64.csv

```diff
@@ -121,7 +121,7 @@ Generic System,Sale Leaseback,756864000,32.130512,35.256783,;
 Generic System,LCOE Calculator,756864000,NA,NA,; 
 Generic System,None,756864000,NA,NA,; 
 PV Battery,Residential,8588.589844,11.205457,NA,; 
-PV Battery,Commercial,911872.062500,4.274726,NA,; 
+PV Battery,Commercial,911877.375000,4.274710,NA,; 
```

Comment:
These are the test values that are in Patch 2. I changed the values with this commit (https://github.com/NREL/SAM/commit/6b8ebd84e106c7b5158b71d49bddbf984fbaa4c9#diff-8b237736d5563a32f81df3035be5cccb0f7b2cfd8b863b1d3d4e5956f65dfb1dR123) and must not have changed it back.